src/Billing/Controllers/StripeController.cs

```diff
@@ -741,5 +746,29 @@ private async Task<Subscription> GetSubscriptionAsync(Stripe.Event parsedEvent,
             }
             return subscription;
         }
+
+        private async Task<Subscription> VerifyCorrectTaxRateForCharge(Invoice invoice, Subscription subscription)
+        {
+            var localBitwardenTaxRates = await _taxRateRepository.GetByLocationAsync(
```

Comment:
Seems you can circumvent this if we don't have a country and/or postal code (if they are `null` or empty) and simply return the `subscription` early in the method. My understanding is "existing" subscriptions may not yet have those kinds of billing details added as they would with the new UI which requires that info (older accounts on auto-renewal)